api/identityserver.proto

```diff
@@ -118,6 +118,14 @@ message IsConfiguration {
     google.protobuf.BoolValue create_organizations = 4;
   }
   UserRights user_rights = 6;
+  message UserLogin {
+    google.protobuf.BoolValue disable_credentials_login = 1;
+  }
+  UserLogin user_login = 7;
```

Comment:
Added user login as well. @pgalic96 pls check missing implementation and backport if necessary.

Reply:
We don't have the option to disable credentials login in Open Source, so this will always be `false`. We can explicitly set a `false` BoolValue in the implementation, but I don't think that's necessary.